.travis.yml

```diff
@@ -18,14 +18,16 @@ cache:
     - vendor
 
 stages:
-  - dep-cache
-  - test
+  - name: dep-cache
+    if: tag != dev
+  - name: test
+    if: tag != dev
   - name: build
-    if: (type != pull_request AND branch = master AND tag IS blank) OR tag IS present
+    if: (type != pull_request AND branch = master AND tag IS blank) OR (tag IS present AND tag != dev)
```

Comment:
Should not we run `build` together with `release`. Would simplify variable hell

Reply:
We are introducing `dev` tag in the `release-dev` stage. When the new tag appears Travis starts a new build for this tag. This condition just prevents this run the second time.